library/src/main/java/org/jellyfin/apiclient/model/dlna/DeviceProfile.java

```diff
@@ -473,16 +473,15 @@ public DeviceProfile()
 
 	public final TranscodingProfile GetAudioTranscodingProfile(String container, String audioCodec)
 	{
-		container = StringHelper.TrimStart((container != null) ? container : "", '.');
-
+		container = (container != null) ? container.replace(".", "") : "";
```

Comment:
This no longer removes leading whitespace. May introduce new issues?

Reply:
I honestly don't think this one will be an issue, do you think so?